sys/linux/dev_img_rogue.txt

```diff
@@ -0,0 +1,1925 @@
+	hComputeContext	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXNOTIFYCOMPUTEWRITEOFFSETUPDATE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETCOMPUTECONTEXTPRIORITY {
+	hComputeContext	IMG_HANDLE
+	ui32Priority	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETCOMPUTECONTEXTPRIORITY {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETCOMPUTECONTEXTPROPERTY {
+	hComputeContext	IMG_HANDLE
+	ui32Property	int32
+	ui64Input	int64
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETCOMPUTECONTEXTPROPERTY {
+	ui64Output	int64
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCURRENTTIME {
+	ui32EmptyStructPlaceholder	const[0, int32]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCURRENTTIME {
+	ui64Time	int64
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFWDEBUGDUMPFREELISTPAGELIST {
+	ui32EmptyStructPlaceholder	const[0, int32]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFWDEBUGDUMPFREELISTPAGELIST {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFWDEBUGPHRCONFIGURE {
+	ui32ui32PHRMode	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFWDEBUGPHRCONFIGURE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFWDEBUGSETFWLOG {
+	ui32RGXFWLogType	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFWDEBUGSETFWLOG {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFWDEBUGSETHCSDEADLINE {
+	ui32RGXHCSDeadline	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFWDEBUGSETHCSDEADLINE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFWDEBUGSETOSIDPRIORITY {
+	ui32OSid	int32
+	ui32Priority	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFWDEBUGSETOSIDPRIORITY {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXFWDEBUGSETOSNEWONLINESTATE {
+	ui32OSid	int32
+	ui32OSNewState	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXFWDEBUGSETOSNEWONLINESTATE {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCONFIGCUSTOMCOUNTERS {
+	ui16CustomBlockID	int16
+	ui16NumCustomCounters	len[pui32CustomCounterIDs, int16]
+	pui32CustomCounterIDs	ptr[in, array[int32]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCONFIGCUSTOMCOUNTERS {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCONFIGENABLEHWPERFCOUNTERS {
+	ui32ArrayLen	len[psBlockConfigs, int32]
+	psBlockConfigs	ptr[in, array[RGX_HWPERF_CONFIG_CNTBLK]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCONFIGENABLEHWPERFCOUNTERS {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCTRLHWPERF {
+	ui32StreamId	int32
+	bToggle		bool32
+	ui64Mask	int64
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCTRLHWPERF {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCTRLHWPERFCOUNTERS {
+	bEnable		bool32
+	ui32ArrayLen	len[pui16BlockIDs, int32]
+	pui16BlockIDs	ptr[in, array[int16]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCTRLHWPERFCOUNTERS {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXGETHWPERFBVNCFEATUREFLAGS {
+	ui32EmptyStructPlaceholder	const[0, int32]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXGETHWPERFBVNCFEATUREFLAGS {
+	sBVNC	RGX_HWPERF_BVNC
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCREATEKICKSYNCCONTEXT {
+	hPrivData		IMG_HANDLE
+	ui32PackedCCBSizeU88	int32
+	ui32ContextFlags	int32
```

Comment:
Same as above. There seem to be some other occurrences of ui32ContextFlags as well.

Reply:
Fixed.